tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -623,3 +633,32 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
             field=None,
         )
     return is_valid, error
+
+
+def t3_child_validator(which_child):
+    """T3 child validator."""
+    def t3_first_child_validator_func(value):
+        if not _is_empty(value, 1, 60) and len(value) >= 60:
+            return (True, None)
+        else:
+            return (False, "1st child record truncated.")
+
+    def t3_second_child_validator_func(value):
+        if not _is_empty(value, 60, 101) and len(value) >= 101 and not _is_empty(value, 8, 19):
+            return (True, None)
+        else:
+            return (False, "2nd child record truncated.")
+
+    return t3_first_child_validator_func if which_child == 1 else t3_second_child_validator_func
+
+
+def is_quiet_preparser_errors(min_length, empty_from=61, empty_to=101):
+    """Return a function that checks if the length is valid and if the value is empty."""
+    def return_value(value):
+        is_length_valid = len(value) >= min_length
+        is_empty = value_is_empty(
```

Comment:
does it make sense to use the `_is_empty` function instead? it calls `value_is_empty` but also performs an additional length check

Reply:
we would need to refactor _is_empty to do what we need here and I think it makes more sense to validate separately